src/main/java/com/minecolonies/coremod/client/gui/ConfigGUI.java

```diff
@@ -0,0 +1,51 @@
+package com.minecolonies.coremod.client.gui;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.minecolonies.coremod.configuration.ConfigurationHandler;
+import com.minecolonies.coremod.lib.Constants;
+
+import net.minecraft.client.gui.GuiScreen;
+import net.minecraft.client.resources.I18n;
+
+import net.minecraftforge.common.config.ConfigElement;
+import net.minecraftforge.fml.client.config.DummyConfigElement;
+import net.minecraftforge.fml.client.config.GuiConfig;
+import net.minecraftforge.fml.client.config.IConfigElement;
+
+/**
+ * Represents the GUI Screen for the in-game GUI.
+ */
+public class ConfigGUI extends GuiConfig 
+{
+    public ConfigGUI(GuiScreen parentScreen) 
+    {
+        super(parentScreen,                                         /* parentScreen */
+                getConfigElement(),                                 /* configElements */ 
+                Constants.MOD_ID,                                   /* modID */
+                true,                                               /* allRequireWorldRestart */
+                true,                                               /* allRequireMcRestart */
+                I18n.format("com.minecolonies.configgui.title"));   /* title */
+    }
+    
+    /** Compiles a list of config elements */
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)